integration-tests/oauth2-reactive/src/test/resources/testRunner.yml

```diff
@@ -43,6 +43,13 @@ scenarios:
     disabledTests:
     # Spring does remote validation of the token
     - invalidIssuerIdTokenJwtTest
+    - expiredIdTokenJwtTest
+    - inactiveTokenJwtTest
+    - invalidSignatureIdTokenJwtTest
+    - issuedInFutureIdTokenJwtTest
+    - unsignedIdTokenJwtTest
+    - wrongAudienceIdTokenJwtTest
+    - wrongKeyIdIdTokenJwtTest
```

Comment:
why are these disabled?

Reply:
Those tests would fail because the Spring tries to validate it remotely. 

I see [this](https://github.com/okta/okta-spring-boot/blob/2da169d7f9d6674504d15b3c41ab0a4e5a0506a0/integration-tests/oauth2-reactive/src/test/resources/testRunner.yml#L42-L45) test already disabled for the same reason.

Below is the error when we leave those tests enabled.

```
[ERROR] expiredIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.125 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.expiredIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:241)

[ERROR] inactiveTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.089 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.inactiveTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:303)

[ERROR] invalidSignatureIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.081 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.invalidSignatureIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:178)

[ERROR] issuedInFutureIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.072 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.issuedInFutureIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:220)

[ERROR] unsignedIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.055 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.unsignedIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:324)

[ERROR] wrongAudienceIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.079 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.wrongAudienceIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:262)

[ERROR] wrongKeyIdIdTokenJwtTest(com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT)  Time elapsed: 0.095 s  <<< FAILURE!
java.net.ConnectException: Connection refused (Connection refused)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:389)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.assertAccessDenied(OIDCCodeFlowLocalValidationIT.groovy:388)
        at com.okta.test.mock.tests.OIDCCodeFlowLocalValidationIT.wrongKeyIdIdTokenJwtTest(OIDCCodeFlowLocalValidationIT.groovy:199)

```

Reply:
@bdemers any followup comments on this?

Reply:
were they passing with the previous version?

Reply:
yes, these tests started failing after we enabled remote validation/opaque token tests.

Reply:
sounds like those would be blocking bugs
can you look into the test failures?

Reply:
Will do.